src/content/en/updates/2017/07/upcoming-regexp-features.md

```diff
@@ -0,0 +1,213 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: This update discusses upcoming features in JavaScript RegExps.
+
+{# wf_updated_on: 2017-07-10 #}
+{# wf_published_on: 2017-07-10 #}
+{# wf_tags: javascript #}
+
+{# wf_featured_image: /web/updates/images/2017/07/v8.png #}
+{# wf_featured_snippet: An overview of the exciting new features coming to
+JavaScript RegExps, including named captures, the dotAll flag, Unicode property
```

Comment:
I’d prefer to stick with `RegExp` too, if only to match the name of the global object in JavaScript.